GameWindow.java

```diff
@@ -442,216 +459,174 @@ public void run() { // it is necessary to check if the program is 'running' or n
 		stop();
 	}
 
-	// This method is for the sole purpose of ensuring buttons remain in sync even
-	// if the user doesn't hit all of the notes
+	
+	
+	
+	//This method is for the sole purpose of ensuring buttons remain in sync even if the user doesn't hit all of the notes
 	public void delayFalling() {
-
+		
 		// FOX WAVEDASH
-		if (FoxWavedash.isSelected()) {
-			if (YnoteFW.getFalling() == false && RnoteFW.getFalling() == false
-					&& stickDLeftnoteFW.getFalling() == false) {
+		if(FoxWavedash.isSelected()) {
+			if (YnoteFW.getFalling() == false && RnoteFW.getFalling() == false && stickDLeftnoteFW.getFalling() == false) {
 				YnoteFW.setFalling(true);
 				RnoteFW.setFalling(true);
 				stickDLeftnoteFW.setFalling(true);
-				counter++;
+				
 			}
 		}
-

```

Comment:
If you've moved the current technique into the `technique` array I described above, you can drastically simplify this bit of code and not have to copy it for new techniques.

```java
boolean anyFalling = false;
for (Note n : technique) {
  if (n.getFalling()) {
    anyFalling = true;
  }
}

if (!anyFalling) {
  for (Note n : technique) {
    n.setFalling(true);
  }
}
```

This removes a lot of fiddly, potentially buggy copy-and-paste as you add new techniques!